From 5ea0eda7618dd041339b15af49e74f703ee8fca4 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Mon, 26 Nov 2012 11:25:55 +0800
Subject: ENGR00234685-2 mx6q_sabreauto: change Sabreauto board to LDO-ENABLED
 mode

Per hardware design, we can't set LDO bypass mode on Sabreauto board,otherwise,
system will can't reset,if cpu freq run in 400Mhz.
Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c b/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
index fe6d4b3..671233f 100644
--- a/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
+++ b/arch/arm/mach-mx6/mx6q_sabreauto_pmic_pfuze100.c
@@ -403,7 +403,7 @@ static int pfuze100_init(struct mc_pfuze *pfuze)
 
 	/*use default mode(ldo bypass) if no param from cmdline*/
 	if (enable_ldo_mode == LDO_MODE_DEFAULT)
-		enable_ldo_mode = LDO_MODE_BYPASSED;
+		enable_ldo_mode = LDO_MODE_ENABLED;
 
 	/*read Device ID*/
 	ret = pfuze_reg_read(pfuze, PFUZE100_DEVICEID, &value);
-- 
1.7.9.5

